src/main/java/emissary/test/core/ExtractionTest.java

```diff
@@ -256,7 +257,27 @@ protected void checkAnswers(Element el, IBaseDataObject payload, List<IBaseDataO
             byte viewData[] = payload.getAlternateView(viewName);
             assertTrue(String.format("Alternate View '%s' is present, but should not be, in %s", viewName, tname), viewData == null);
         }
+    }
 
+    public void checkAttachmentAndExtractedRecordCount(Element el, IBaseDataObject payload, List<IBaseDataObject> attachments, String tname)
+            throws DataConversionException {
+        int numAtt = JDOMUtil.getChildIntValue(el, "numAttachments");
+        int attFoundCount = 0;
+        if (numAtt > -1) {
+            int attCount = 1;
```

Comment:
You should not assume that the att count starts at one. Test will pass if you add att2 element.